docs/readthedocs/models/input/grid/transformer3w.md

```diff
@@ -178,9 +178,67 @@ All impedances and admittances are given with respect to the higher voltage side
 
 ```
 
+## Standard Three Winding Transformer Types
+

```

Comment:
Could you add an input file for this type input? We did this for the standard line and two winding transformer types.